### ISSUE TITLE
Proxy

### ISSUE THREAD

**Issue by DKanarsky** [ramusus/kinopoiskpy#36]
Есть способ задать прокси для отправки запросов с разных ip адресов. Известно ли какой лимит и штрафные санкции?

**Comment by ramusus** [comment]
Нет, не известно. После начала использования сессий, я не видел больше капчи в ответах

**Comment by DKanarsky** [comment]
Опытным путем было выяснено, что get_content('main_page') срабатываем около полусотни раз, после чего ip жестко банится на сутки. Если действовать через тор, то довольно часто появляется каптча, плюс многие точки выхода уже забанены, то есть похоже, что ведется какой-то black list ip, но это не точно.

**Comment by ramusus** [comment]
тогда нужно прикручивать прокси. я бы предпочел бы это сделать опционально подключаемым модулем

**Comment by DKanarsky** [comment]
Можно дать возможность передавать в качестве параметра уже настроенный на прокси request

**Comment by theshadowagent** [comment]
Похожая проблема, начал выдавать капчу при вызове get_content('main_page'), только мой IP вроде еще не забанен (с браузера ввел капчу один раз, и теперь работает), и при отдельном вызове requests.get('kinopoisk.ru') капчу не выдает. Много запросов не делал. Пробовал менять константу HEADERS, не помогает

**Comment by theshadowagent** [comment]
Заработало когда изменил фильм. Видимо, капча вылезает при нескольких запросах к одному и тому же фильму

**Comment by fatklass** [comment]
> тогда нужно прикручивать прокси. Я бы предпочел бы это сделать отдельно подключаемым опциональным модулем. В самом простом случае `requests` позволяет это делать через переменную окружения http://docs.python-requests.org/en/master/user/advanced/#proxies

На данный момент функцию для работы через прокси так и не реализовали?
